README.md

```diff
@@ -12,11 +13,33 @@ yarn install
 yarn build
 ```
 
+## Development
+To run on local, run
+```
+yarn start
+```
+
+This will run the app with local data (not API call) located in `lib/json`. This folder is ignored in the repo. You can create your own by downloading from Firebase console (Database Tab) those records:
```

Comment:
@mamadOuologuem let's create some mock data and let people know that they can replace it in `'lib/json`